### DISCUSSION TITLE
Using an AssetPostprocessor?

### DISCUSSION THREAD

**Discussion by frnsys** [atteneder/glTFast#727] [Q&A]
Is it possible for an AssetPostprocessor to hook into importing events e.g. `OnPostprocessModel` or not? I've defined an AssetPostprocessor but it doesn't seem to react to the GLTF importing process.

**Comment by atteneder** [comment]
I personally have not tried to pull that off, but @cprkv did in this [comment/Gist](https://github.com/atteneder/glTFast/issues/731#issuecomment-2506726173).
